### DISCUSSION TITLE
Process jobs from dynamically created queues

### DISCUSSION THREAD

**Discussion by Guidanz-Kishore** [OptimalBits/bull#1942] [Q&A]
## Description
I have a usecase where I need to create queues during runtime and the queue names are variables (For eg: <userId>-queue).
I would like to know how we can specify process functions for these dynamically created queues. Can you please let me know what is the recommended way of doing this?
## Minimal, Working Test code to reproduce the issue.
```js
const createUser=()=>{
let userId=createUserObj();
const queue=new Queue(`${userId}-queue`);
queue.process('namedProcess',concurrency, processFunc);//processFunc() is the processing function defined in the same file.
queue.close();
}
const processFunc = (job) => {
...
}
```
## Bull version
3.18.1

**Comment by Guidanz-Kishore** [comment]
@manast Should i post this in "Issues" category? Am i doing something wrong?

**Comment by manast** [comment]
Issues should be for bugs or feature requests. Now that github supports "Discussions" it is a better place for these kind of questions.
